packages/sdk/src/sdk/api/generated/default/apis/CommentsApi.ts

```diff
@@ -42,9 +42,9 @@ export class CommentsApi extends runtime.BaseAPI {
      * @hidden
      * Gets replies to a parent comment
      */
-    async getCommentRepliesRaw(params: GetCommentRepliesRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<CommentResponse>> {
+    async commentRepliesRaw(params: CommentRepliesRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<CommentResponse>> {
```

Comment:
I'm curious why the change in name here?

Reply:
I just noticed the other content methods didn't have a get before then, what should we do?

Reply:
Consistency is good. Though it does mean updating usage in client.

Reply:
Oh also we probably need a changeset